tests/test_import_export.py

```diff
@@ -17,6 +17,21 @@
 from tests.data import expected as expected_data, input as input_data
 
 
+def _normalize_xml(dirname):
+    """Removes whitespace from xml files in the given directory."""
+    for dname, _, files in os.walk(dirname):
+        for fname in files:
+            fpath = os.path.join(dname, fname)
+            if not fpath.endswith('.xml'):
+                continue
+            with open(fpath) as f:
+                s = f.read()
```

Comment:
This seems kind of hacky to me since it doesn't ignore spaces or newlines within elements. Why is Travis not pretty printing in the same way as the Dockerfile included? If you ignore `pp_xml` and use `etree` to pretty print we should probably get consistent results across platforms